config-model/pom.xml

```diff
@@ -11,14 +11,61 @@
 
     <artifactId>config-model</artifactId>
 
+    <properties>
+        <kafka-metadata-version>${kafka.version}</kafka-metadata-version>
+        <config-model-file>../cluster-operator/src/main/resources/kafka-${kafka-metadata-version}-config-model.json</config-model-file>
+    </properties>
+
+    <dependencyManagement>
+        <!-- Manage versions to allow for override via the CLI with system property. -->
```

Comment:
Can you add to the comment the why we need to be able to override with system props?